File: Python/URI/1015.py
import math

linha1 = input().split(" ")
linha2 = input().split(" ")

x1, y1 = linha1
x2, y2 = linha2

distancia = math.sqrt(((float(x1)-float(x2))**2)+((float(y1)-float(y2))**2));

print("%0.4f" %distancia)

File: JS/URI/1015.js
const input = require('fs').readFileSync('stdin', 'utf8');
const lines = input.split('\n');

const [x1, y1] = lines[0].split(' ');
const [x2, y2] = lines[1].split(' ');

const ponto = Math.sqrt(((x2-x1)**2 + (y2-y1)**2));

console.log(ponto.toFixed(4));